Tools/clinic/clinic.py

```diff
@@ -1187,11 +1205,17 @@ def output_templates(
             docstring_prototype = docstring_definition = ''
         elif f.kind is GETTER:
             methoddef_define = self.GETTERDEF_PROTOTYPE_DEFINE
-            docstring_prototype = docstring_definition = ''
+            if f.docstring:
+                docstring_prototype = self.GETSET_DOCSTRING_PROTOTYPE_VAR
+                docstring_definition = self.GETSET_DOCSTRING_PROTOTYPE_STRVAR
+            else:
+                docstring_prototype = docstring_definition = ''
         elif f.kind is SETTER:
+            if f.docstring:
+                fail("@setter can not set docstring")
```

Comment:
I'd tweak the error message so the user gets a hint about how the clinic input should have been written.

```suggestion
                fail("docstrings are only supported for @getter, not @setter")
```

```suggestion
                fail("only @getter can provide docstrings")
```